### PR TITLE
Added additional tests & checks

### PR DESCRIPTION
**Pull request by M0RG4N01**
Added additional tests & checks

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,26 +3,33 @@
...
-        throw new Error ("enter a valid value");
-    }
-    return Math.min(...arrVal);
-} 
+const minVal = arrVal => {
+
+  if(arrVal == null) {
+    throw new Error("Error: Parameter is empty")
+  }
+  else if(arrVal.length === 0) {
```
Comment (missKatiaPunter) [inline index.js:11]: I like it, well done @M0RG4N01 👍

File: index.js [hunk 2]
```diff
@@ -3,26 +3,33 @@
...
 // Does it have a logic error?
 // Refactor and test some more
 
 const fizzBuzz = n => {
     let result = [];
-    for(let i=1;i<n+1;i++){
-        if(i%5===0 && i%3===0){
-            result.push('fizzbuzz');
-        } else if(i%3===0){
+    let number = 15;
```
Comment (missKatiaPunter) [inline index.js]: Wow! Good coding!

File: index.spec.js [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
+
+    it('Should return a value of data type number', () => {
         expect(typeof minVal([1,2,34,0,-12])).toBe('number');
         expect(minVal([1,2,34,0,-12])).not.toBeInstanceOf(Object);
     });
-  
+
     it('Should get a minimal value from an array', () => {
-        expect(minVal([1,2,34,0,-12])).toBe(-12);
+        expect(minVal([2, -12])).toBe(-12);
```
Comment (missKatiaPunter) [inline index.spec.js]: I am not sure this improves the readability of tests. Was that the reason for this change?

**Review by missKatiaPunter — CHANGES_REQUESTED** [review]

File: index.spec.js (hunk 3 above)
Comment (M0RG4N01) [reply index.spec.js]: Resolved ,given better test data

**Review by missKatiaPunter — APPROVED** [review]

**Pull request merged**